en/orm.rst

```diff
@@ -31,13 +31,17 @@ database connections <database-configuration>`.
 Quick Example
 =============
 
-To get started you don't have to write any code. If you've followed the CakePHP
-conventions for your database tables you can just start using the ORM. For
-example if we wanted to load some data from our ``articles`` table we could do::
+To get started you don't have to write any code. If you've followed the :doc:`CakePHP
+conventions for your database tables </intro/conventions.html#model-and-database-conventions>`
```

Comment:
Hi ravage84. I read over the RST documenation to make sure I understood which to use. I probably goofed. I'll go back and review it again.  Thanks!